src/mscorlib/shared/Interop/Unix/Interop.Libraries.cs

```diff
@@ -8,5 +8,6 @@ internal static partial class Libraries
     {
         internal const string GlobalizationInterop = "System.Globalization.Native";
         internal const string SystemNative = "System.Native";
+        internal const string LibCoreCLR = "libcoreclr";
```

Comment:
Here is what you need to do:
* Your windows p/invoke can be shared between CoreCLR and CoreRT
* For Unix, put your coreclr p/invoke under #ifdef CORECLR, and #else throw PNSE